File: WebBiblioteca_Api/BibliotecaApiDLL/editora/Editora.cs
namespace BibliotecaApiDLL.editora
{
    public class Editora
    {
        public Editora(int codEditora, string nomeEditora, string descricaoEDitora)
        {
            CodEditora = codEditora;
            NomeEditora = nomeEditora;
            DescricaoEDitora = descricaoEDitora;
        }

        public int CodEditora { get; set; }
        public string NomeEditora { get; set; }
        public string DescricaoEDitora { get; set; }
    }
}

File: WebBibliotecaApp/src/Objects/Editora.ts
export class Editora {
    codEditora = 0;
    nomeEditora: string = '';
    descricaoEDitora: string = ''

    constructor(codEditora: number, nomeEditora: string, descricaoEDitora: string ){
        this.codEditora = codEditora;
        this.nomeEditora = nomeEditora;
        this.descricaoEDitora = descricaoEDitora;
    }
}
